ballerina/Module.md

```diff
@@ -156,6 +156,43 @@ string|int? generatedKey = result.lastInsertId;
 These samples show how to demonstrate the different usages of the `query` operation to query the
 database table and obtain the results. 
 
+The `ParameterizedQuery` is used to construct the dynamic query to be executed by the client. You can create a simple query like below.
```

Comment:
IMO this section should be separated from querying data. Shall we have a new section called parameterized queries and explain this